mapproxy/cache/tile.py

```diff
@@ -76,6 +78,7 @@ def __init__(self, grid, cache, sources, format, locker, image_opts=None, reques
         self.sources = sources
         self.minimize_meta_requests = minimize_meta_requests
         self._expire_timestamp = None
+        self._refresh_before = {}
```

Comment:
how this var gets populated from config?

Reply:
It is set in [loader.py#L1578](https://github.com/mapproxy/mapproxy/blob/65d05bf34ba4a0928043b0dfc1963e40e3ae332a/mapproxy/config/loader.py#L1578)
This is not very pretty, but is a similar fashion as for mapproxy-seed which also externally sets this attribute af the tile manager:
- in seed - refresh: [seeder.py#L512](https://github.com/camptocamp/mapproxy/blob/refresh_while_serving/mapproxy/seed/seeder.py#L512)
- in seed- cleanup: [cleanup.py#L116](https://github.com/camptocamp/mapproxy/blob/refresh_while_serving/mapproxy/seed/cleanup.py#L116)